lib/maplibre-compose/src/commonMain/kotlin/dev/sargunv/maplibrecompose/compose/controls/AttributionDialog.kt

```diff
@@ -0,0 +1,66 @@
+package dev.sargunv.maplibrecompose.compose.controls
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.material3.AlertDialogDefaults
+import androidx.compose.material3.BasicAlertDialog
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalUriHandler
+import androidx.compose.ui.semantics.heading
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.unit.dp
+import dev.sargunv.maplibrecompose.compose.StyleState
+
+@Composable
+@OptIn(ExperimentalMaterial3Api::class)
+public fun AttributionDialog(
+  styleState: StyleState,
+  onDismiss: () -> Unit,
+  titleText: String = "Data sources",
+  buttonText: String = "OK",
```

Comment:
I'd say, this library needs translations, now. (Maybe the Android library already has some, which can be copied into this project?)

When the text is properly translated, I don't even think it makes sense to expose these parameters anymore.

Reply:
Would it be easy for the consuming app to provide translations for languages the library itself doesn't support? If so then yeah, agreed. Having them as parameters is just a quick hack to provide that flexibility now

Reply:
I don't know the compose resource system, but in Android resource system, it wouldn't be possible.